Type error with @types/react >=18.2.66

**Describe the bug**

Type-checking `@use-gesture/react` with `@types/react@>=18.2.66` (and `skipLibCheck` turned off) results in an error:

```
./node_modules/@use-gesture/react/dist/declarations/src/types.d.ts(4,33): error TS2536: Type 'Key' cannot be used to index type 'DOMAttributes<EventTarget>'.
```

**Sandbox or Video**

N/A

**Information:**
 - Use Gesture version: v10.3.0
 - Device: N/A
 - OS: N/A
 - Browser N/A

**Checklist:**
  - [x] I've read the [documentation](https://use-gesture.netlify.app/).
  - [x] If this is an issue with drag, I've tried setting `touch-action: none` to the draggable element.

--- a/.changeset/stale-seahorses-speak.md
+++ b/.changeset/stale-seahorses-speak.md
@@ -1,0 +1,5 @@
+---
+"@use-gesture/core": patch
+---
+
+fix(types): compatibility with @types/react >=18.2.66

--- a/packages/core/src/types/utils.ts
+++ b/packages/core/src/types/utils.ts
@@ -165,9 +165,7 @@ export interface DOMHandlers {
   onPointerCancel?: EventHandler<PointerEvent>
   onPointerCancelCapture?: EventHandler<PointerEvent>
   onPointerEnter?: EventHandler<PointerEvent>
-  onPointerEnterCapture?: EventHandler<PointerEvent>
   onPointerLeave?: EventHandler<PointerEvent>
-  onPointerLeaveCapture?: EventHandler<PointerEvent>
   onPointerOver?: EventHandler<PointerEvent>
   onPointerOverCapture?: EventHandler<PointerEvent>
   onPointerOut?: EventHandler<PointerEvent>

